src/main/resources/application.yml

```diff
@@ -1,14 +1,23 @@
 nextflow:
+  cluster:  #context to determine the cluster the workflow will run in.
+    masterUrl:
+      azure: "localhost:8080/azure"
+      cumulus: "localhost:8080/cumulus"
+    context:
+      azure: "azure-context-name"
+      cumulus: "cumulus-context-name"
+    volMounts:
+      azure:
+        - "pv-claim:/some/dir/in/azure"
+      cumulus:
+        - "pv-claim:/some/dir/in/cumulus"
```

Comment:
The nesting order feels inverted here, would it be better to have the cluster properties grouped together?

```yml
cluster:  #context to determine the cluster the workflow will run in.
    azure:
        masterUrl: "localhost:8080/azure"
        context: "azure-context-name"
        volMounts: 
          - "pv-claim:/some/dir/in/azure"
    cumulus:
        masterUrl: "localhost:8080/cumulus"
        context: "cumulus-context-name"
        volMounts: 
          - "pv-claim:/some/dir/in/cumulus"
```

Reply:
I really like providing two clusters in the built in application.yml so that the infra team has an example for how to configure multiple clusters. However, I don't want to commit an example that is so closely tied to the setup of a specific RDPC. Remembering that this is designed to run in multiple RDPCs, the example should include a cluster setup to run with the docker-compose that is committed to this repo, and any provided test examples should work with that default. 

A second cluster should maybe just be commented out so we can see how to configure it but not have it included in the config when this is deployed to a production environment.